torch/distributed/elastic/multiprocessing/tail_log.py

```diff
@@ -111,10 +113,13 @@ def start(self) -> "TailLog":
             return self
 
         for local_rank, file in self._log_files.items():
+            header = f"[{self._name}{local_rank}]:"
+            if self._log_line_headers and local_rank in self._log_line_headers:
```

Comment:
Is the intention to use the default if an empty dict is passed? can we document that behavior?